short_answer/static/js/src/short_answer.js

```diff
@@ -1,6 +1,13 @@
 /* Javascript for ShortAnswerXBlock. */
 function ShortAnswerXBlock(runtime, element, options) {
+  const $answerSubmitButton = $('.answer-submission-button', element);
+  const $shortAnswer = $('.short-answer', element);
+  const gradesPublished = options.gradesPublished;
   const maxScore = options.maximumScore;
+  const passedDue = options.passedDue;
+  const score = options.score;
+
```

Comment:
nitpick: no need to have this empty line.

Reply:
There is no need for this line but I wanted to separate variables which depend on other variables so it's easier to locate them. I would like to leave it as is and include a comment.